<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWindow</class>
 <widget class="QMainWindow" name="DeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>324</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>341</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ITUSB1 USB Test Switch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxControl">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>101</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="pushButtonAttach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDetach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Detach</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPower">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Power</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLCDNumber" name="lcdNumberCurrent">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>61</y>
       <width>79</width>
       <height>29</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurrent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>67</y>
       <width>105</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current (mA)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Display</string>
     </property>
     <widget class="QRadioButton" name="radioButtonDisplayAct">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Actual</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonDisplayAvg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Average</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonDisplayMin">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>35</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Min.</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonDisplayMax">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>65</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max.</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelOCFault">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>217</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <addaction name="actionInformation"/>
    <addaction name="separator"/>
    <addaction name="actionSaveData"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuUpdateRate">
     <property name="title">
      <string>&amp;Update Rate</string>
     </property>
     <addaction name="actionRate50"/>
     <addaction name="actionRate100"/>
     <addaction name="actionRate200"/>
     <addaction name="actionRate300"/>
     <addaction name="actionRate500"/>
    </widget>
    <addaction name="menuUpdateRate"/>
    <addaction name="actionPersistent"/>
    <addaction name="separator"/>
    <addaction name="actionLogData"/>
    <addaction name="actionDeleteData"/>
    <addaction name="actionResetTime"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRate100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;100ms</string>
   </property>
  </action>
  <action name="actionRate200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;200ms</string>
   </property>
  </action>
  <action name="actionRate300">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;300ms</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>&amp;Information</string>
   </property>
  </action>
  <action name="actionPersistent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Persistent Warnings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About ITUSB1 Manager</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save Logged Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLogData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Log Data</string>
   </property>
  </action>
  <action name="actionDeleteData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Logged Data</string>
   </property>
  </action>
  <action name="actionRate50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5&amp;0ms</string>
   </property>
  </action>
  <action name="actionRate500">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;500ms</string>
   </property>
  </action>
  <action name="actionResetTime">
   <property name="text">
    <string>&amp;Reset Time Count</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonAttach</tabstop>
  <tabstop>pushButtonDetach</tabstop>
  <tabstop>checkBoxPower</tabstop>
  <tabstop>checkBoxData</tabstop>
  <tabstop>pushButtonReset</tabstop>
  <tabstop>radioButtonDisplayAct</tabstop>
  <tabstop>radioButtonDisplayAvg</tabstop>
  <tabstop>radioButtonDisplayMin</tabstop>
  <tabstop>radioButtonDisplayMax</tabstop>
  <tabstop>pushButtonClear</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>DeviceWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
